apps/wing-console/console/ui/src/features/map-view.tsx

```diff
@@ -682,7 +682,10 @@ export const MapView = memo(
         return (
           <ConstructNode
             id={props.constructTreeNode.path}
-            name={props.constructTreeNode.id}
+            name={
```

Comment:
@skyrpex just making sure you tested:
display.title, "as", and ID 
for reference:
https://github.com/winglang/wing/pull/6211
https://github.com/winglang/wing/issues/5994